content/design-principles.en.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Design principles"
+---
+
+# Design principles
+
+These overarching principles guide technical and governance decisions. They are fundamental and can only be changed through community consensus, based on a thorough impact assessment.
+
+Each principle has a name, a rationale, and potential implementation examples or guidelines.
+
+## 1. Never trust the services
+
+A major goal of Open Terms Archive is to enable assessing the loyalty of services towards their end users. Since loyalty is not assumed, trust can not be warranted.
```

Comment:
Maybe "loyalty" towards end users is not the best term here? Maybe better "transparency" or "trustworthiness"?

Reply:
This term was chosen after several weeks of work at @AmbNum. We found it was the broadest term that was still meaningful and objective to mean “having their best interest in mind”.
“Transparency” is restricted to telling what you do, not on not behaving like assholes. Open Terms Archive does enable assessing behavior, not only transparency.
“Trustworthiness” might be closer to the idea of “having best interest in mind”, but is hard to understand and needlessly hard to translate in our (and most international institutions’) second working language, French. “loyalty”, on the other hand, translates as “loyauté”.